chapters/publication.tex

```diff
@@ -268,94 +266,91 @@ \section{Preparing a complete replication package}
 if not, paring it down to the ``replication package'' may take some time.
 A complete replication package should accomplish several core functions.
 It must provide the exact data and code that is used for a paper,
-all necessary de-identified data for the analysis,
+all necessary (de-identified) data for the analysis,
 and all code necessary for the analysis.
-The code should exactly reproduce the raw outputs you have used for the paper,
-and should not include documentation or data you would not share publicly.
+The code and data should exactly reproduce the raw outputs you have used for the paper,
+and the replication file should not include any documentation or data you would not share publicly.
 This usually means removing project-related documentation such as contracts
 and details of data collection and other field work,
 and double-checking all datasets for potentially identifying information.
 
 \subsection{Publishing data for replication}
 
-Enabling permanent access to the data used in your study
-is an important contribution you can make along with the publication of results.
-It allows other researchers to validate the mechanical construction of your results,
-to investigate what other results might be obtained from the same population,
-and test alternative approaches or other questions.
-Therefore you should make clear in your study
-where and how data are stored, and how and under what circumstances they may be accessed.
-You do not always have to publish the data yourself,
-and in some cases you are legally not allowed to.
+Publicly documenting all original data generated as part of a research project 
+is an important contribution in its own right.
+Your paper should clearly cite the data used,
+where and how it is stored, and how and under what circumstances it may be accessed.
+You may not be able to publish the data itself, 
+due to licensing agreements or ethical concerns.
 Even if you cannot release data immediately or publicly,
-there are often options to catalog or archive the data without open publication.
+there are often options to catalog or archive the data.
 These may take the form of metadata catalogs or embargoed releases.
 Such setups allow you to hold an archival version of your data
 which your publication can reference,
-as well as provide information about the contents of the datasets
+and provide information about the contents of the datasets
 and how future users might request permission to access them
-(even if you are not the person who can grant that permission).
+(even if you are not the person to grant that permission).
 They can also provide for timed future releases of datasets
 once the need for exclusive access has ended.
 
 If your project collected original data,
 releasing the cleaned dataset is a significant contribution that can be made
 in addition to any publication of analysis results.\sidenote{
```

Comment:
This seem repetitive with the previous paragraph. Maybe we remove the first three lines and move the citation?

Reply:
addressed in 4a75b156edd45d4059ada18a5ce30972f7256a7f